src/ContentHub.php

```diff
@@ -583,4 +583,62 @@ protected function getResponseJson(RequestInterface $request)
         $body = (string) $response->getBody();
         return json_decode($body, TRUE);
     }
+
+    /**
+     * Fetch snapshots.
+     *
+     * @return mixed
+     *   Response.
+     *
+     * @throws \Exception
+     */
+    public function getSnapshots()
+    {
+      return self::getResponseJson($this->get('snapshots'));
```

Comment:
I am not sure how can this work. getResponseJson() function first parameter should be a RequestInterface. $this->get('snapshots') will return a ResponseInterface